app/services/hyrax/versioning_service.rb

```diff
@@ -57,12 +57,15 @@ def perform_create_through_active_fedora(content, user)
         end
 
         def perform_create_through_valkyrie(content, user)
+          return # TODO: WINGS - Just return for now.  This method won't work until #indexing_adapter method is complete.  See issue #3800.
```

Comment:
Just skip this method for now as it's pending test is failing and needs further exploration when this method is completed.